### ISSUE TITLE
Running without Docker

### ISSUE THREAD

**Issue by scharc** [subspacecloud/subspace#15]
What is the right way to set it up without Docker?

**Comment by kapilt** [comment]
it looks like compile the binary (oddly there's is one checked into the repo), and install the following programs iproute2 iptables dnsmasq socat  .. per the dockerfile. also odd is the use of phusion base image instead of a std distro image.

**Comment by subspacecloud** [comment]
Yes @kapilt is correct, you need to do the same things that the Docker image does to make it work. I will add instructions to the README in the future so it's more clear. Thanks for the report.

**Comment by Fastidious** [comment]
@scharc @kapilt @subspacecloud there seems to be a lot of things that need to be done before being able to run the executable (non-docker), based on `entrypoint.sh`.

**Comment by Shagon94** [comment]
seconding this, theres quite a lot of things to perform in the `entrypoint.sh`, a more streamlined guide without docker would be great

**Comment by kawaiiDango** [comment]
How do I compile this? I am trying to run without docker and my port 80 ans 443 are already in use, so I cant use that binary without compiling.

**Comment by kawaiiDango** [comment]
I tried to run without docker, but go build produces errors
./mailer.go:91:27: undefined: AssetNames
./mailer.go:96:13: undefined: Asset
./web.go:109:27: undefined: AssetNames
./web.go:114:13: undefined: Asset
./web.go:253:12: undefined: Asset
./web.go:258:13: undefined: AssetInfo

**Comment by dooferlad** [comment]
Seems like the current rev is missing a file containing AssetNames, Asset, AssetInfo.

**Comment by sklamt** [comment]
You need to run go-bindata to generate the missing assets . 

In the Dockerfile the following commands are used to generate and validate the assets:
`
go-bindata --pkg main static/... templates/... email/...
go fmt
go vet --all
`

**Comment by bmullan** [comment]
So has anyone gotten a NON-Docker version to build yet?  If so are there any instructions that could be posted here?

thanks !

**Comment by Xipherisroot** [comment]
Going to give this a poke since I'm VERY interested in this project but without using Docker at all.

I'm really confused why there isn't instructions that aren't "get the Docker file" !?

**Comment by storrgie** [comment]
I'll also pile on.
